chat/be/server.js

```diff
@@ -242,6 +242,7 @@ async function run() {
         {
           $set: {
             body: newBody,
+            edited: true,
```

Comment:
Can we change this to updatedAt and make it a Date?  That way we also are able to store when the message was last edited